schematics/ng-componize/utils.ts

```diff
@@ -1,46 +1,151 @@
-import { strings, join, basename, normalize } from '@angular-devkit/core';
-import { buildDefaultPath } from '@schematics/angular/utility/project';
-import { FileEntry } from '@angular-devkit/schematics';
-import { getProject } from '@schematics/angular/utility/project';
-import { WorkspaceSchema } from '@schematics/angular/utility/workspace-models';
+import { join, normalize, strings } from '@angular-devkit/core';
+import { Tree } from '@angular-devkit/schematics';
 import { parseName } from '@schematics/angular/utility/parse-name';
+import {
+  buildDefaultPath,
+  getProject,
+} from '@schematics/angular/utility/project';
+import {
+  ProjectType,
+  WorkspaceProject,
+  WorkspaceSchema,
+} from '@schematics/angular/utility/workspace-models';
+import * as fs from 'fs';
+import * as Parser from './HtmlParser';
+import { extractParameterName } from './htmlParser';
```

Comment:
these are two types of imports from the same file, It's better to merge them 
import { func1, func2, extractParameterName } from './htmlParser'; and get rid of import * as Parser from './HtmlParser';